src/cmd/deploy_nsfs.sh

```diff
@@ -0,0 +1,416 @@
+provisioner: kubernetes.io/no-provisioner
+volumeBindingMode: WaitForFirstConsumer
+";
+
+echo "$storage_class_yaml" | kubectl apply -f -;
+
+print_done
+
+# create nsfs persistent volume
+echo -e "Creating NSFS local persistent volume:\n";
+
+pv_yaml="
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+    name: nsfs-vol
+spec:
+    storageClassName: nsfs-local
+    volumeMode: Filesystem
+    persistentVolumeReclaimPolicy: Retain
+    local:
+        path: /nsfs/
+    capacity:
+        storage: 1Ti
+    accessModes:
+        - ReadWriteMany
+    nodeAffinity:
+        required:
+            nodeSelectorTerms:
+                - matchExpressions:
+                    - key: kubernetes.io/os
+                      operator: Exists
+";
+echo "$pv_yaml" | kubectl apply -f -;
+
+print_done
+
+
+# create nsfs persistent volume claim
+echo -e "Creating NSFS local persistent volume claim:\n";
+
+pvc_yaml="
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+    name: nsfs-vol
+spec:
+    storageClassName: nsfs-local
+    resources:
+        requests:
+            storage: 1Ti
+    accessModes:
+        - ReadWriteMany
+";
+echo "$pvc_yaml" | kubectl apply -f -;
+print_done;
+}
+
+
+# delete_sc_pv_pvc deletes the nsfs pv, pvc and storageclass
+function delete_sc_pv_pvc(){
+print_headline "NSFS RESOURCES DELETION" "Deleting NSFS local storage class:"
+kubectl delete pv nsfs-vol --wait=false;
+kubectl patch pv nsfs-vol --type json --patch='[ { "op": "remove", "path": "/metadata/finalizers" } ]'
+kubectl delete pvc nsfs-vol --wait=false;
+kubectl patch pvc nsfs-vol --type json --patch='[ { "op": "remove", "path": "/metadata/finalizers" } ]'
+kubectl delete storageclass nsfs-local;
+print_done
+}
+
+
+# minikube_export_dir_creation logins to minikube host and creates the file system path will be used 
+function minikube_export_dir_creation(){
+print_headline "MINIKBE DIR CREATION" "Creating nsfs/bucket-path/ dir and changing its permissions to 777:"
+minikube ssh -- "sudo mkdir -p /nsfs/bucket-path/; sudo chmod -R 777 /nsfs/bucket-path/;";
+print_done;
+}
+
+
+# minikube_export_dir_deletion- logins to minikube host and deleted the file system path used by the nsfs resource
+function minikube_export_dir_deletion(){
+print_headline "MINIKBE DIR DELETION" "Deleting nsfs/bucket-path/ dir:"
+minikube ssh -- "sudo rmdir /nsfs/bucket-path/;";
+print_done
+}
+
+
+# create_namespace_resource - creates the noobaa nsfs namespacestore on top of the file system pvc
+# both exported bucket and s3 bucket will be on top of this namespace resource
+function create_namespace_resource(){
+print_headline "NAMESPACE RESOURCE CREATION" "Creating nsfs namespacestore on top of a pvc"
+noobaa namespacestore create nsfs fs1 --pvc-name='nsfs-vol';
+print_done;
+}
+
+
+# delete_namespace_resource - deleted the nsfs namespacestore
+function delete_namespace_resource(){
+print_headline "NAMESPACE RESOURCE DELETION" "Deleting nsfs namespacestore on top of a pvc:"
```

Comment:
In the other headlines, e didn't use colon
```suggestion
print_headline "NAMESPACE RESOURCE DELETION" "Deleting nsfs namespacestore on top of a pvc"
```